{
    "type": "bundle",
    "id": "bundle--daf41f64-b342-4284-bbf3-05585c46aec3",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--7b32dbe8-9813-4a8e-a82f-450efa745d12",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-05-02T00:28:12.799075Z",
            "modified": "2024-05-02T00:28:12.799075Z",
            "name": "CVE-2024-27047",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\nnet: phy: fix phy_get_internal_delay accessing an empty array\n\nThe phy_get_internal_delay function could try to access to an empty\narray in the case that the driver is calling phy_get_internal_delay\nwithout defining delay_values and rx-internal-delay-ps or\ntx-internal-delay-ps is defined to 0 in the device-tree.\nThis will lead to \"unable to handle kernel NULL pointer dereference at\nvirtual address 0\". To avoid this kernel oops, the test should be delay\n>= 0. As there is already delay < 0 test just before, the test could\nonly be size == 0.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-27047"
                }
            ]
        }
    ]
}